script.js

```diff
@@ -1,12 +1,32 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let images = [];
+let mySound;
+
+function preload() {
+  soundFormats("mp3", "ogg");
+  mySound = loadSound("StarsAndStripesForever.mp3");
+  img1 = loadImage("fireworks1.png");
```

Comment:
you got lucky here, but you actually need to declare all these images with `let img1 = ...` etc.